### ISSUE TITLE
Octo and OpenVLA fine-tuned checkpoint on LIBERO

### ISSUE THREAD

**Issue by ZhixuLi-the-Runner** [RoboDita/Dita#3]
Dear author:
Nice work and beautiful figures! If you don't mind, would you like to share the code of your baseline on LIBERO, e.g. the octo and openvla? There are some works that also take Octo on LIBERO as a baseline, but I didn't see their code implementation. I believe it definitely can elevate your work's impact if you release your implementation of octo, openvla on LIBERO.

Best regards

**Comment by ZTY0213** [comment]
Hello,
Thanks for your interest! I have updated a script about finetuning Libero, please read the [README.md](https://github.com/RoboDita/Dita/blob/main/README.md) and try to use it. Feel free to contact us if you still have any questions!

Best

**Comment by ZhixuLi-the-Runner** [comment]
> Hello, Thanks for your interest! I have updated a script about finetuning LIBERO, please read the [README.md](https://github.com/RoboDita/Dita/blob/main/README.md) and try to use it. Feel free to contact us if you still have any questions!
> 
> Best

Dear author:
Thank you for the reply, I've noticed that in https://github.com/RoboDita/Dita/issues/5#issuecomment-2769717995 you mentioned, existing pre-trained model is already good enough for general inference (e.g. LIBERO Long), and then you will give a checkpoint ( fine-tuned ) on LIBERO, which is exciting! Looking forward to that!

**Comment by zhihou7** [comment]
Hi @ZhixuLi-the-Runner, If you use this [pretraining checkpoint](https://drive.google.com/file/d/1a3nkXCieQL7GaemZH_NFpXYNd_LO3_ZU/view?usp=sharing), you can get around 78%. This checkpoint is pretrained on Droid only. Interesting, we find pretraining on Droid is more beneficial for Libero, while pretraining on OXE is more beneficial for Calvin(ABC->D). Here is a [libero-long finetuned checkpoint](https://drive.google.com/file/d/1vA7BJ57mcu4pPo-QxgM8plzh6UfdmXyj/view?usp=drive_link)

**Comment by ZhixuLi-the-Runner** [comment]
> Hi [@ZhixuLi-the-Runner](https://github.com/ZhixuLi-the-Runner), If you use this [pretraining checkpoint](https://drive.google.com/file/d/1a3nkXCieQL7GaemZH_NFpXYNd_LO3_ZU/view?usp=sharing), you can get around 78%. This checkpoint is pretrained on Droid only. Interesting, we find pretraining on Droid is more beneficial for Libero, while pretraining on OXE is more beneficial for Calvin(ABC->D). Here is a [libero-long finetuned checkpoint](https://drive.google.com/file/d/1vA7BJ57mcu4pPo-QxgM8plzh6UfdmXyj/view?usp=drive_link). I will update the code with eval_only and README soon.
> 
> For reproducing the results, you should use `scheduler_type=0 num_inference_steps=10`.
> 
> ` torchrun --nproc_per_node=8 --nnodes=1 --node_rank=0 scripts/train_diffusion_oxe.py task_name=finetuning_LIBERO dataname=libero_spatial_no_noops dataset.traj_length=11 num_pred_action=10 scheduler_type=0 shuffle_buffer_size=128000 batch_size=64 use_close_loop_eval=True +trajectory_dim=7 +pretrained_path=dit_policy_checkpoint.pth +use_adjust_scheduler=true lr=0.0001 +min_lr_scale=0.01 +image_aug=true num_inference_steps=10`
> 
> Best,
Dear author,
Thank you for the checkpoint and evaluation script! And one more thing is the libero-fine-tuned checkpoint is not public accessible, I submitted several requests and still not get the permission yet

**Comment by zhihou7** [comment]
Sorry for forgetting the permission. I have made it public.
